### PR TITLE
Types for npm package: react-hooks-helper

### PR DESCRIPTION
**Pull request by jedmundo**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4296,6 +4296,7 @@
 /types/react-highlight-words/                       @mhegazy @diogodca @kellyrmilligan
 /types/react-highlighter/                           @oizie
 /types/react-holder/                                @isman-usoh
+/types/react-hot-loader/                            @jedmundo
```
Comment (armanio123) [inline .github/CODEOWNERS]: Don't change this file as is auto-generated. Instead you can be added on the react-hot-loader "definititions by" if you'll like to be a code-owner.

File: types/react-hooks-helper/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+// Type definitions for react-hooks-helper 1.6
```
Comment (armanio123) [inline types/react-hooks-helper/index.d.ts]: I noticed that this package also distributes a umd module. You might want to add a namespace for that. `export as namespace myLibl;`.

Check [this](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html) template for details.

**Review by armanio123 — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (jedmundo) [reply .github/CODEOWNERS]: Thanks! Done.

File: types/react-hooks-helper/index.d.ts (hunk 2 above)
Comment (jedmundo) [reply types/react-hooks-helper/index.d.ts]: done!

**Review by armanio123 — APPROVED** [review]

**Pull request merged**